src/cli.zig

```diff
@@ -1797,10 +1797,7 @@ pub const Command = struct {
             const file = bun.toLibUVOwnedFD(((brk: {
                 if (std.fs.path.isAbsolute(script_name_to_search)) {
                     var win_resolver = resolve_path.PosixToWinNormalizer{};
-                    var resolved = win_resolver.resolveCWD(script_name_to_search) catch @panic("Could not resolve path");
-                    if (comptime Environment.isWindows) {
-                        resolved = resolve_path.normalizeString(resolved, true, .windows);
-                    }
+                    const resolved = win_resolver.resolveCWD(script_name_to_search) catch @panic("Could not resolve path");
```

Comment:
can you check that `bun-debug (run) C:\path\to\..\file.js` works? I think that's why normalizeString was here before

Reply:
Didn't think of that. Makes me think we should normalize this string on all platforms.

Node does path.resolve on the argv[1]:
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/dtCe81PMjdXXNBWQ43OR/0421ed39-9985-4b21-8db2-dfd4bd866d84.png)

Reply:
im going to revert this and just fix it in a followup